sentry_sdk/integrations/strawberry.py

```diff
@@ -291,13 +293,17 @@ def _patch_execute():
     old_execute_sync = strawberry_schema.execute_sync
 
     async def _sentry_patched_execute_async(*args, **kwargs):
-        # type: (Any, Any) -> ExecutionResult
+        # type: (Any, Any) -> Union[ExecutionResult, SubscriptionExecutionResult]
         result = await old_execute_async(*args, **kwargs)
 
         if sentry_sdk.get_client().get_integration(StrawberryIntegration) is None:
             return result
 
-        if "execution_context" in kwargs and result.errors:
+        if "execution_context" in kwargs and (
+            result.errors
+            if isinstance(result, ExecutionResult)
+            else await async_any(r.errors async for r in result)
+        ):
```

Comment:
We can simplify this and remove the check for `errors` altogether and always add the event processor -- we also do it that way in other integrations (i.e., we don't check for errors beforehand).

```suggestion
        if "execution_context" in kwargs:
```

Same in the sync wrapper on line 318. Then we don't need this special handling or the new util.

Reply:
Nice idea, I will implement this